docs/versioning.rst

```diff
@@ -0,0 +1,156 @@
+**********
+Versioning
+**********
+
+
+Current Release Stages
+======================
+
+.. list-table:: Releases
+   :widths: 20 30 30 20
+   :header-rows: 1
+
+   * - Release
+     - Runtimes
+     - Minimum Trace Agent
+     - Stage
+   * - 0.x
+     - CPython: 2.7, 3.5–3.10
+     - 
+     - MAINTENANCE [2022-03-01]
+   * - 1.x
+     - CPython: 2.7, 3.5+
+     - 7.28
+     - CURRENT
+
+
+Runtimes
+========
+
+
+.. list-table:: Supported runtime versions
```

Comment:
Is there a reason this is centered in the page? do we control that?

<img width="644" alt="image" src="https://user-images.githubusercontent.com/1320353/160618075-855d687d-4f35-48d4-96da-2af1e8de674a.png">

Reply:
👍🏻 then ignore, thanks!